integritybackend/claim.py

```diff
@@ -305,7 +386,7 @@ def _get_location_created(self, lat, lon):
 
         return self._remove_keys_with_no_values(location_created)
 
-    def _make_exif_data(self, meta):
+    def _make_exif_data(self, lat: int, lon: int, timestamp: str):
```

Comment:
ts from different sources come in different formats. I'll make some normalization logic later.